uv.lock

```diff
@@ -1,5 +1,5 @@
 version = 1
-revision = 1
+revision = 2
```

Comment:
To note, I did not change anything in uv.lock. They just bumped the version of it and this is the reason why there are so many changes

I did `uv sync` to update mcstatus to 12.0 inside uv.lock